### PR TITLE
Convert some internal only functions to async await

### PR DESCRIPTION
**Pull request by Page-**

### CODE REVIEW

File: src/sbvr-api/sbvr-utils.ts [hunk 1]
```diff
@@ -1489,183 +1468,174 @@ const runGet = (
...
 
-	return runQuery(tx, request, undefined, idField)
-		.tap(sqlResult => {
-			if (sqlResult.rowsAffected === 0) {
-				throw new PermissionError();
-			}
-			return validateModel(tx, vocab, request);
-		})
-		.then(({ insertId }) => insertId);
+	const sqlResult = await runQuery(tx, request, undefined, idField);
```
Comment (dfunckt) [inline src/sbvr-api/sbvr-utils.ts]:
```suggestion
	const { insertId } = await runQuery(tx, request, undefined, idField);
```

File: src/sbvr-api/sbvr-utils.ts [hunk 2]
```diff
@@ -1489,183 +1468,174 @@ const runGet = (
...
+	if (sqlResult.rowsAffected === 0) {
+		throw new PermissionError();
+	}
+	await validateModel(tx, vocab, request);
+
+	return sqlResult.insertId;
```
Comment (dfunckt) [inline src/sbvr-api/sbvr-utils.ts]:
```suggestion
	return insertId;
```

File: src/sbvr-api/sbvr-utils.ts [hunk 3]
```diff
@@ -1489,183 +1468,174 @@ const runGet = (
...
 };
 
-const runPut = (
+const runPut = async (
 	_req: _express.Request,
 	_res: _express.Response,
 	request: uriParser.ODataRequest,
 	tx: _db.Tx,
-): Bluebird<undefined> => {
+): Promise<undefined> => {
```
Comment (dfunckt) [inline src/sbvr-api/sbvr-utils.ts]:
```suggestion
): Promise<void> => {
```

?

File: src/sbvr-api/sbvr-utils.ts [hunk 4]
```diff
@@ -1489,183 +1468,174 @@ const runGet = (
...
 const respondOptions = respondPut;
 
-const runDelete = (
+const runDelete = async (
 	_req: _express.Request,
 	_res: _express.Response,
 	request: uriParser.ODataRequest,
 	tx: _db.Tx,
-) => {
+): Promise<undefined> => {
```
Comment (dfunckt) [inline src/sbvr-api/sbvr-utils.ts]:
```suggestion
): Promise<void> => {
```

?

File: src/sbvr-api/sbvr-utils.ts (hunk 3 above)
Comment (Page-) [reply src/sbvr-api/sbvr-utils.ts]: The `undefined` is specifically used elsewhere, basically these `runX` functions can return `_db.Result | number | undefined`

**Pull request merged**